dl/pom.xml

```diff
@@ -239,4 +240,27 @@
             </plugin>
         </plugins>
     </reporting>
+    
+    <profiles>
+    	<profile>
+    		<id>parallel-tests</id>
+   			<properties>
+      			<runSpecsInParallel>true</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Serial</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-tests</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+      			<tagsToExclude>com.intel.analytics.bigdl.tags.Parallel</tagsToExclude>
+   			</properties>
+		</profile>
+		<profile>
+    		<id>serial-all</id>
+   			<properties>
+      			<runSpecsInParallel>false</runSpecsInParallel>
+   			</properties>
+		</profile>
+    </profiles>
```

Comment:
So is it means two maven commands "mvn test -Pparallel-tests" and "mvn test -Pserial-tests" to be run when we want to execute the maven test? Is it possible to combine them to one command?